esx_service/cli/vmdkops_admin.py

```diff
@@ -619,24 +623,15 @@ def get_vmdk_size_info(path):
     Get the capacity and used space for a given VMDK given its absolute path.
     Values are returned as strings in human readable form (e.g. 10.00MB)
 
-    Currently this data is retrieved via a call to vmkfstools.
-    The output being parsed looks like the following:
-
-    Capacity bytes: 209715200
-    Used bytes: 27262976
-    Unshared bytes: 27262976
+    Using get_vol_info api from volume kv. The info returned by this
+    api is in human readable form
     """
     try:
-        cmd = "vmkfstools --extendedstatinfo {0}".format(path).split()
-        output = subprocess.check_output(cmd)
-        result = output.decode('utf-8')
-        lines = result.split('\n')
-        capacity_in_bytes = lines[0].split()[2]
-        used_in_bytes = lines[1].split()[2]
-        return {'capacity': human_readable(int(capacity_in_bytes)),
```

Comment:
this method has references in ```generate_tenant_access_ls_rows``` method in the same file
The references are as below. 

```
        # p[auth_data_const.COL_MAX_VOLUME_SIZE] is max_volume_size in MB
        max_vol_size = UNSET if p.max_volume_size == 0 else human_readable(p.max_volume_size * MB)
        # p[auth_data_const.COL_USAGE_QUOTA] is total_size in MB
        total_size = UNSET if p.usage_quota == 0 else human_readable(p.usage_quota * MB)
```